config/settings/production.py

```diff
@@ -33,14 +33,7 @@
 # Make sure SecurityMiddleware is listed first
 MIDDLEWARE_CLASSES = SECURITY_MIDDLEWARE + MIDDLEWARE_CLASSES
 
-# set this to 60 seconds and then to 518400 when you can prove it works
-SECURE_HSTS_SECONDS = env.int('DJANGO_SECURE_HSTS_SECONDS', default=60)
-SECURE_HSTS_INCLUDE_SUBDOMAINS = True
-SECURE_CONTENT_TYPE_NOSNIFF = True
-SECURE_BROWSER_XSS_FILTER = True
-SESSION_COOKIE_SECURE = True
```

Comment:
I'm not sure what is the benefit from removing these from the Django settings. I'm not totally against it, but having them in the app configuration does at least force some good practises to people deploying the app. In cases where Django is able to affect something, after all it depends on the web server setup too.

However, `SESSION_COOKIE_SECURE` I'm pretty sure shouldn't be removed. It's not something afaict that can be set on web server side. If removal is to make sure http sites can be run - I don't think we should be encouraging that.

What is the main point for removing the settings?